src/expr/scalar/func.rs

```diff
@@ -1748,6 +1770,11 @@ impl BinaryFunc {
                 _ => unreachable!(),
             },
 
+            RoundDecimal(_) => {
+                // The output type can be null because `round` may cause arithmetic overflow.
```

Comment:
Can `round` still cause arithmetic overflow if we are following PostgreSQL behavior and not T-SQL?

Reply:
No, now null can only be caused by null of parameters.